Check broken links (#1354)
ocaml/v3.ocaml.org: data/changelog/merlin/2014-08-23-merlin-2.0.0.md

  * automatic reloading of .merlin files (when they are update or created), it
    is no longer necessary to restart Merlin
  * introduced a small refactoring command: rename, who renames all occurences

>     of an identifier. See [here](http://yawdp.com/~host/merlin_rename.webm - [1 Client error: Couldn't resolve host name]).

Reviewer: ```suggestion
    of an identifier. See [here](https://github.com/ocaml/merlin/blob/master/vim/merlin/doc/merlin.txt).
```
This is the only thing close I could find.

Reviewer 2: I don't think that's what they were linking to, so I'm removing the link altogether.

```suggestion
    of an identifier.
```

---

After a few months of development, we are pleased to announce the

> [stable release](https://github.com/the-lambda-church/merlin/blob/master/CHANGELOG - [404 Not Found]) of

Reviewer: ```suggestion
[stable release](https://github.com/ocaml/merlin/blob/master/CHANGES.md) of
```

Reviewer 2: They're announcing the stable release of Merlin 2.0.0. This is not technically the same as the latest version of Merlin. If we're keeping the link, it makes more sense to link to the changelog of Merlin 2.0.0.

Reviewer 2: ```suggestion
[stable release](https://github.com/ocaml/merlin/blob/v2.0/CHANGELOG) of
```
